extension/docs/DevelopmentAndDeploymentGuide.md

```diff
@@ -51,15 +53,28 @@ There are 3 different types of tests. Don't forget to provide all required envir
 with the UI (e.g. credit card 3ds). In such cases, we use Cypress.io. Run `npm run cypress-ui` to test.
 
 ## Deployment
+
+### AWS Lambda
+
+For deployment to lambda zip the extensions folder and specify `src/lambda.handler` as the entry point for the function
+
+When deploying the lambda, it will not create the custom types for you. These are required for the extension to operate correctly.
```

Comment:
```suggestion
When deploying the lambda, it will NOT create the custom types for you. These are required for the extension to operate correctly. Please add [payment custom types](../resources/payment-custom-types.json) and [payment interface interaction types](../resources/payment-interface-interaction-types.json) manually.
```

Reply:
@zackpollard can you make this small change and check if the links I provided are correct? Then I will approve this PR and merge.

Reply:
Done!